src/styles/card.css

```diff
@@ -0,0 +1,154 @@
+.card {
+  max-width: var(--card-max-width);
+  background-color: var( --card-background-color);
+  border: var(--card-border) solid var(--border-color);
+  border-radius: var(--card-border-radius);
+  box-sizing: border-box;
+}
+
+.card__header {
+  box-sizing: border-box;
+  width: 100%;
+  margin: var(--card-margin);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding-left: 19px;
+  padding-right: 19px;
+  padding-top: var(--card-header-top-gap);
```

Comment:
You can just use shorthand property
https://developer.mozilla.org/en-US/docs/Web/CSS/padding#syntax